app/templates/index.hbs

```diff
@@ -19,9 +19,11 @@
               </li>
             {{/if}}
             <li class="nav-item">
-              <LinkTo @route="index" @query={{hash author=null tag=null page=1}} class="nav-link {{unless this.tag "active"}}">
-                Global Feed
-              </LinkTo>
+              {{#let (and (not this.tag) (not this.author)) as |isGlobalFeed|}}
+                <LinkTo @route="index" @query={{hash author=null tag=null page=1}} class="nav-link {{if isGlobalFeed "active"}}">
```

Comment:
Because the `Global Feed` tab loses its `active` highlighting if the `page` is a non-`1` number, I've opted some `let` logic to calculate when the `active` state for this link should be.